### PR TITLE
ACM-3108 add env var to search deployments

### PR DESCRIPTION
**Pull request by SherinV**
### Related Issue

https://issues.redhat.com/browse/ACM-3108

### Description of changes
- Pass env variables to deployments controlled by the search operator.

### CODE REVIEW

File: controllers/create_pgconfigmap.go
Comment (SherinV) [inline controllers/create_pgconfigmap.go:23]: We use this in line 46 to create work_memquery.
work_memquery := "psql -d search -U searchuser -c \"ALTER ROLE searchuser set work_mem='" + work_mem + "'\""
Do we need to do this for other variables that the user adds?

Comment (SherinV) [reply controllers/create_pgconfigmap.go:23]: Discussed in scrum and agreed that this will be a separate user story.

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold

Before merging, we need to confirm that the CRD updates were propagated to search-v2-operator-bundle and picked up by the installer.

**Comment by SherinV** [comment]
> /lgtm /hold
> 
> Before merging, we need to confirm that the CRD updates were propagated to search-v2-operator-bundle and picked up by the installer.

Installer picked up changes in https://github.com/stolostron/multiclusterhub-operator/pull/1015

**Pull request merged**
